.kilocode/rules/superpowers-session-handoff.md

```
@@ -0,0 +1,67 @@
+1. Next step 1
+2. Next step 2
+
+### Files modified:
+- `src/file1.py`
+- `src/file2.py`
+
+### Commands to continue:
+```bash
+command to run
+```
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)